Sources/Public/iOS/Compatibility/CompatibleAnimationView.swift

```diff
@@ -28,11 +28,17 @@ public final class CompatibleAnimation: NSObject {
     super.init()
   }
 
+  public init(animation: LottieAnimation) {
+    name = ""
+    subdirectory = nil
+    bundle = Bundle.main
+    super.init()
+    self.animation = animation
+  }
+
   // MARK: Internal
 
-  var animation: LottieAnimation? {
-    LottieAnimation.named(name, bundle: bundle, subdirectory: subdirectory)
-  }
+  private(set) lazy var animation = LottieAnimation.named(name, bundle: bundle, subdirectory: subdirectory)
```

Comment:
Instead, what if we just make this:

```suggestion
  var animation: LottieAnimation
```

and update the existing `init` to just:

```swift
  public init(
    name: String,
    subdirectory: String? = nil,
    bundle: Bundle = Bundle.main)
  {
    animation = LottieAnimation.named(name, bundle: bundle, subdirectory: subdirectory)
    super.init()
  }
```

Reply:
No problem